_reports/RohanVB.md

```diff
@@ -0,0 +1,263 @@
+   - details: >
+      Initializes SublimeLinter-coala without proper regex
+
+ - 19:
+   - repo: SublimeLinter-coala
+   - link: https://github.com/RohanVB/SublimeLinter-coala/commit/8a2b9d531c7344df32287eb36419e71671c423c3
+   - details: >
+      Fixes regex and brings SublimeLinter-coala to working state
+
+ - 20:
+   - repo: coala-syntastic
+   - link: https://github.com/RohanVB/coala-syntastic/commit/15c7e00b52ffed4e3e9f59ede29d883981ed8549
+   - details: >
+      Fixes version parsing
+
+ - 21:
+   - repo: coala-syntastic
+   - link: https://github.com/RohanVB/coala-syntastic/commit/adca116e30fdd9464ce79a17c2d5842f0fff1484
+   - details: >
+      Removes unrequired checkers
+
+ - 22:
+   - repo: coala-syntastic
+   - link: https://github.com/RohanVB/coala-syntastic/commit/a03b2253695df6dc11e9b187bca0a746b4d4a13a
+   - details: >
+      Fixes regex issue
+
+ - 23:
+   - repo: coala-syntastic
+   - link: https://github.com/RohanVB/coala-syntastic/commit/de8a9e9556c69a0c833489908073889d2a8ea8df
+   - details: >
+      Fixes error format
+
+ - 24:
+   - repo: coala-syntastic
+   - link: https://github.com/RohanVB/coala-syntastic/commit/b6922bdf15ea9d2ac61d2aee183b0cd152d2761f
+   - details: >
+      Fixes format for some bears
+
+ - 25:
+   - repo: coala-syntastic
+   - link: https://github.com/RohanVB/coala-syntastic/commit/d927715f602c43061dd4fb21948e09fc06049192
+   - details: >
+      Fixes error format
+
+ - 26:
+   - repo: coala-syntastic
+   - link: https://github.com/RohanVB/coala-syntastic/commit/d927715f602c43061dd4fb21948e09fc06049192
+   - details: >
+      Fixes error format
+
+ - 27:
+   - repo: syntastic-coala
+   - link: https://github.com/RohanVB/syntastic-coala/commit/b0e9adcad6a13b41f0d2a2c31c1c8caf51ac49d1
+   - details: >
+      Restructures repository and adds installation instructions
+
+
+ - 28:
+   - repo: dependency_management
+   - link: https://gitlab.com/coala/package_manager/commit/918d24b32408ffea7d70b71796bf14c846870177
+   - details: >
+      Adds AtomRequirement
+
+ - 29:
+   - repo: dependency_management
+   - link: https://gitlab.com/coala/package_manager/commit/b4e7c06a6d78caef78ee2cf20bec11fbddb72a0f
+   - details: >
+      Adds tests for AtomRequirement
+---
+
+### Cohesive Editor Support
+
+
+#### Work Done
+
+[coala spyder](https://gitlab.com/coala/editors/coala-spyder) is now a functional coala plugin and can be launched from the spyder editor. The repository has unittests and continuous integration set up with TravisCI.
+It can also be tested as a stand-alone widget.
+
+<br>
+
+[coala sublime](https://gitlab.com/coala/editors/coala-sublime) is now a functional plugin for `Sublime Text 3` with all of the bells and whistles of SublimeLinter.
+[coala vim](https://gitlab.com/coala/editors/coala-vim) is now a functional plugin for `vim` and `neovim` powered by the functionality of `syntastic`.
+
+<br>
+
+The three above projects have been added to the main `coala` repository.
```

Comment:
repository -> "organisation on GitLab, which involved rewriting the commit history to remove the fixups, and meeting the requirements of an maintainable organisational project."